### ISSUE TITLE
Bug with AmiraHyperSurfaceSegmentation

### ISSUE THREAD

**Issue by aliaksei-chareshneu** [emdb-empiar/sfftk#25]
Dear all,

There is a bug with AmiraHyperSurfaceSegmentation:
```
app_spec_seg = AmiraHyperSurfaceSegmentation(filepath_str)
```
`filepath_str` is valid filename

```
Exception has occurred: ValueError will not overwrite attribute 'name'
```

**Comment by paulkorir** [comment]
Fixed in upcoming `v0.6.0.dev0`.

**Comment by paulkorir** [comment]
This is an exception that originates from `ahds`, the Python package used to read Amira files. I will close this here but open an issue in the `ahds` repository. Could you please give me a bit more details about the file? Perhaps if you can share it with me. You can find the issue at https://github.com/emdb-empiar/ahds/issues.
